libs/api/domains/license-service/src/lib/client/driving-license-client/genericDrivingLicense.config.ts

```diff
@@ -0,0 +1,51 @@
+import { defineConfig } from '@island.is/nest/config'
+import * as z from 'zod'
+
+export const schema = z.object({
+  xroad: z.object({
+    baseUrl: z.string(),
+    clientId: z.string(),
+    path: z.string(),
+    secret: z.string(),
+  }),
+  pkpass: z.object({
+    apiKey: z.string(),
+    apiUrl: z.string(),
+    secretKey: z.string(),
+    cacheKey: z.string(),
+    cacheTokenExpiryDelta: z.string(),
+    authRetries: z.string(),
+  }),
+})
+
+export const GenericDrivingLicenseConfig = defineConfig<z.infer<typeof schema>>(
+  {
+    name: 'GenericDrivingLicenseConfig ',
+    schema,
+    load: (env) => ({
+      xroad: {
+        baseUrl: env.required('XROAD_BASE_PATH', 'http://localhost:8081'),
+        clientId: env.required(
+          'XROAD_CLIENT_ID',
+          'IS-DEV/GOV/10000/island-is-client',
+        ),
```

Comment:
We generally import the xroad base path and client id from the shared XRoadConfig.

You may consider creating new shared configuration for other stuff if it makes sense (pkpass?), but no stress.